### ISSUE TITLE
Improve rating radio buttons

### ISSUE THREAD

**Issue by oliverjam** [oliverjam/shitty-vegan#2]
Currently choosing a rating uses standard radios. These could look a lot nicer, and be bigger tap targets.

@jamchiller do you have any design thoughts?

**Comment by Jamchiller** [comment]
Will design these up today

**Comment by Jamchiller** [comment]
Done! 
![rating buttons](https://user-images.githubusercontent.com/16238581/44330128-25a12900-a45e-11e8-9757-b28ce71a4d31.png)

**Comment by oliverjam** [comment]
Wondering about the copy here: do we want to use the technical terms or something more friendly like "Ate no meat", "Ate eggs", "Ate dairy" etc

**Comment by Jamchiller** [comment]
Yeah I remember we had this convo back in Feb. It's a hard one but maybe we should leave it as is now and can test later with both? I can't think of good titles...
- Completely vegan
- Ate eggs
- Ate eggs and dairy
- Ate meat

Kinda works but it hurts the association with the mascots a lil. Happy either way

**Comment by oliverjam** [comment]
Can still keep the mascots next to the radios. Not sure you have to include eggs _and_ dairy in the veggie one. If you do "Ate eggs" and "Ate dairy" I can't see how they'd fill in the wrong one.

Definitely something to test

**Comment by Jamchiller** [comment]
Let's put it on the backlog to test for now. I'm worried that if someone ate eggs and dairy (as most do), they'll be unsure of which one to tap. Our hierarchical system (ie, that eating eggs is ethically better than eating dairy) is definitely not something that is clear to everyone at this point. Ovo-vegetarian makes the hierarchy clear

**Comment by oliverjam** [comment]
Good point. I guess the colours help?

**Comment by oliverjam** [comment]
In progress. Slight problem: the way the popover library does the arrows makes it basically impossible to give them a box-shadow. Tbh I wanted to completely re-implement myself anyway, but that will be a huge job to build something that's always pointing at the right place

![localhost_3000_ pixel 2](https://user-images.githubusercontent.com/9408641/44428676-0e148e00-a58d-11e8-8b5d-57a06ffad680.png)

**Comment by Jamchiller** [comment]
All good, that looks absolutely fine for MVP! We'll backlog it

**Comment by Jamchiller** [comment]
Put them in there now!
